### DISCUSSION TITLE
&FULLPATH Action variable for workspace files

### DISCUSSION THREAD

**Discussion by RobBoeckermann** [codefori/vscode-ibmi#561] [Q&A] [answered]
When running an Action on a streamfile in ILE, I am able to use the variable &FULLPATH for the "Full path of the file on the remote system". When running the same action action on a Local File (Workspace), is there any way to have the &FULLPATH variable represent the path the local file is being deployed to on the remote system prior to running the action?

This is the Action I would like to run:
{
    "type": "streamfile",
    "extensions": ["GLOBAL"],
    "environment": "ile",
    "name": "Compile C++",
    "command": "CRTSQLCPPI OBJ(&CURLIB/&NAME) SRCSTMF('&FULLPATH')[...],
    "deployFirst": true
  },

**Comment by worksofliam** [comment, marked as answer]
It looks like `&FULLPATH` should work when `deployFirst` is used, but there is a bug with it. It will be fixed in 1.0.1.

**Reply by worksofliam** [reply]
Released

**Reply by RobBoeckermann** [reply]
Thanks, Liam! It's working now.
